components/builder-jobsrv/src/server/handlers.rs

```diff
@@ -615,47 +615,6 @@ pub fn job_group_get(
     Ok(())
 }
 
-pub fn job_graph_package_create(
```

Comment:
I think we still need this in some fashion, otherwise the graph is not going to get extended when new packages are added (uploaded)..

Reply:
The simplest thing could be to just take the sync_packages function and run it from within the main loop of the scheduler thread.. you can then put the graph extension code in there

Reply:
yeah you're right, let me think about this some more